terraform/pagerduty/member-services-integrations.tf

```diff
@@ -2052,12 +2052,24 @@ resource "pagerduty_service" "sprinkler-development" {
   alert_creation          = "create_alerts_and_incidents"
 }
 
-resource "pagerduty_service_integration" "sprinkler-integration" {
-  name    = data.pagerduty_vendor.cloudwatch.name
-  service = pagerduty_service.sprinkler-development.id
-  vendor  = data.pagerduty_vendor.cloudwatch.id
+resource "pagerduty_team" "sprinkler-development" {
+  name = "sprinkler-development"
+}
+
+resource "pagerduty_event_orchestration" "monitor-sprinkler-integration" {
+  depends_on = [ pagerduty_team.sprinkler-development  ]
+  name = "My Monitoring Orchestration of Sprinkler"
```

Comment:
I think this orchestration can be left here for the team to use to test future pagerduty changes.  Could you name it and the description something to reflect that, eg name="sprinkler-development-integration" description="Integrates sprinkler-development account with PagerDuty" thanks

Reply:
I have amended the description to the one you suggested.

Reply:
You have put the description in the name field and the description is still the same

Reply:
Corrected